divio_cli/cli.py

```diff
@@ -415,19 +415,34 @@ def project_push():
 
 @project_push.command(name='db')
 @click.argument('stage', default='test')
-@click.option('--noinput', is_flag=True, default=False, help="Don't ask for confirmation")
+@click.option(
```

Comment:
you can use Click's builtin file type with checking paths ([see here](https://github.com/divio/divio-cli/blob/master/divio_cli/cli.py#L458))